Package.swift

```diff
@@ -51,6 +51,9 @@ let package = Package(
         "KSCrashRecording",
         "KSCrashRecordingCore",
       ],
+      resources: [
+        .process("Resources"),
```

Comment:
Why not `.copy`?

Reply:
Didn't work for me. But maybe I missed something. Could you try this as I'm not very into that hack with the bundle.